src/tallies/filter_meshsurface.cpp

```diff
@@ -19,7 +19,7 @@ void MeshSurfaceFilter::get_all_bins(
 
   Direction u = p.u();
   model::meshes[mesh_]->surface_bins_crossed(r0, r1, u, match.bins_);
-  for (auto b : match.bins_)
+  for (unsigned int b = 0; b < match.bins_.size(); ++b)
```

Comment:
This seems kind of silly -- the variable is unused in the body of the loop either way, and the change here doesn't result in better code